eq-author/src/App/QuestionnairesPage/QuestionnairesTable/Row.test.js

```diff
@@ -60,51 +61,109 @@ describe("Row", () => {
     expect(focus).toHaveBeenCalled();
   });
 
-  it("should allow duplication of a Questionnaire", () => {
-    const wrapper = shallow(
-      <Row
-        questionnaire={questionnaire}
-        onDeleteQuestionnaire={handleDeleteQuestionnaire}
-        onDuplicateQuestionnaire={handleDuplicateQuestionnaire}
-      />
-    );
+  it("should handle row focus state change correctly", () => {
+    const wrapper = shallow(<Row questionnaire={questionnaire} {...props} />);
+    const tableRow = wrapper.find(TR);
+
+    expect(wrapper.state("linkHasFocus")).toBeFalsy();
```

Comment:
Can we write this test without relying on state that maybe checking for a style or something as it feels really internal and doesn't actually prove it does anything.

Reply:
Done.